manual/manual/tutorials/coreexamples.etex

```diff
@@ -27,6 +27,8 @@ by ";;" in response to the "#" prompt, and the system compiles them
 on the fly, executes them, and prints the outcome of evaluation.
 Phrases are either simple expressions, or "let" definitions of
 identifiers (either values or functions).
+The ";;" is not necessary and generally not used in source files created
+for use with the OCaml compiler.
```

Comment:
I think this sentence would fit much better in the "Standalone Ocaml programs" section.

Reply:
I'll move it.  It seemed important to me to include so early because I've been looking at a lot of OCaml code in files.  (Tangentially, many of the examples are far longer than you would ever type in to the top-level.  In practice you would create files for them.)

The standalone OCaml program example at the end of page 24 uses ";;", which I think is not needed and so maybe should be removed.

Reply:
> It seemed important to me to include so early because I've been looking at a lot of OCaml code in files

I agree that this an important point to keep in mind, at the same time this part of the manual is really focused on the interactive toplevel. Maybe a good option would be to add a forward reference or a footnote to the standalone program section for readers more interested in files.

> The standalone OCaml program example at the end of page 24 uses ";;", which I think is not needed and so maybe should be removed.

This is an excellent idea!

Reply:
I can remove two of the ";;".  The one after "exit" is needed (gives a compile error without it).  (Still figuring out OCaml syntax.)

```
let rec fib n =
  if n < 2 then 1 else fib (n-1) + fib (n-2)
let main () =
  let arg = int_of_string Sys.argv.(1) in
  print_int (fib arg);
  print_newline ();
  exit 0;;
main ()
```

Reply:
A common OCaml idiom is to replace  `;; main ()` with `let () = main ()`. 
An interpretation of the syntactical rule for `;;` is that `;;` is used to start (effectful) toplevel expressions.